src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => number;
+}
 
+function makeRoundedNumber(number: number): number {
+  return Math.floor(number * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  readonly shape = Shape.Triangle;
 
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (
+      a <= 0
+      || b <= 0
+      || c <= 0
```

Comment:
Create a helper function that accepts any number of args and checks them to be > 0. Reuse it with all shapes.